src/raygun.js

```diff
@@ -646,6 +646,10 @@ var raygunFactory = function (window, $, Raygun, undefined) {
             }
         }
 
+        if (Raygun.Utilities.isReactNative() && !Raygun.Utilities.contains(options.tags), 'React Native') {
```

Comment:
Good spotting there, much appreciate the report. Apologies for the inconvenience, I'm deploying a `v2.6.0-SNAPSHOT.5` release with your fix however as you may have found out the mainline v2.5.3 release is stable if you prefer to use that.